Whiskey/Functions/New-WhiskeySemanticVersion.ps1

```diff
@@ -39,6 +44,21 @@ function New-WhiskeySemanticVersion
     {
         $semVersion = $Version | ConvertTo-WhiskeySemanticVersion -ErrorAction Stop
     }
+    elseif( $Path )
+    {
+        if($Path -like '*/package.json')
+        {
+            $semVersion = Get-Content -Raw -Path $Path | 

```

Comment:
Add validation that you got a version back. If you didn't, the `package.json` is either invalid JSON, or doesn't contain a version property. `ConvertTo-Json` throws an exception when JSON is bad, so you'll need to add a test for that situation.

Write an error, e.g. `Write-Error -Message ('Unable to determine module version from ''{0}''. Please make sure this is valid JSON, that it contains a Version property, and that the Version is a valid semantic version, e.g. it follows the specification at http://semver.org.' -f $fileInfo.FullName)

Remove the `-ErrorAction Stop` on the `ConvertTo-WhiskeySemanticVersion` so that we can write a more helpful error message.

Reply:
Done